### ISSUE TITLE
Clarification: default port for connections

### ISSUE THREAD

**Issue by sveneh** [ethereum/pyethereum#194]
running `pyeth -h` gives

```
 -p REMOTE_PORT, --port REMOTE_PORT
                        <port> Connect to remote port (default: 30303)
```

but when pyeth actually tries to connect, without setting the remote port explicitly, it does
```
DEBUG:net	connecting failed	error=timeout('timed out',) host='54.164.25.143' port=30300
```
over port 30300

which is the default port for connections? I assume 30303 ? Either the help message is wrong or the code.

**Comment by heikoheiko** [comment]
the bootstrapping currently node listens to 30300, therefore in the default config the remote port is set as 30300

**Comment by heikoheiko** [comment]
fixed in 058903f
